src/status_im2/contexts/shell/cards/view.cljs

```diff
@@ -190,8 +190,8 @@
      [rn/view {:style (style/base-container color-50)}
       (when banner
         [rn/image
-         {:source (:source banner)
-          :style  {:width 160}}])
```

Comment:
Please also confirm changes related to quo2 components in quo2 preview screens. This change will breaks banner in the preview

Reply:
In preview screen, `banner` value is boolean, not image source.